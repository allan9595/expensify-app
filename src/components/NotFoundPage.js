import React from 'react';
import { Link, NavLink } from 'react-router-dom';
const NotFoundPage = () => (
  <div>
    404! - <Link to="/">Go Home</Link>
  </div>
);
export default NotFoundPage;
