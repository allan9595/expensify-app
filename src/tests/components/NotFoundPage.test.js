import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should get the NotFoundPage out correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
