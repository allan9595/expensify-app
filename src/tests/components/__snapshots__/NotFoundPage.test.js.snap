// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get the NotFoundPage out correctly 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
